superset-frontend/src/dashboard/components/OverwriteConfirm/OverwriteConfirmModal.tsx

```diff
@@ -0,0 +1,212 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React, { useMemo, useCallback, RefObject, createRef } from 'react';
+import moment from 'moment';
+import { useDispatch } from 'react-redux';
+import ReactDiffViewer from 'react-diff-viewer';
+import { useInView } from 'react-intersection-observer';
+import Modal from 'src/components/Modal';
+import Button from 'src/components/Button';
+import { DashboardState } from 'src/dashboard/types';
+import {
+  saveDashboardRequest,
+  setOverrideConfirm,
+} from 'src/dashboard/actions/dashboardState';
+import { t, styled } from '@superset-ui/core';
+import { SAVE_TYPE_OVERWRITE_CONFIRMED } from 'src/dashboard/util/constants';
+
+const STICKY_HEADER_TOP = 16;
+const STICKY_HEADER_HEIGHT = 32;
+
+const StyledTitle = styled.h2`
+  ${({ theme }) => `
+     color:  ${theme.colors.grayscale.dark1}
+   `}
+`;
+
+const StyledEditor = styled.div`
+  ${({ theme }) => `
+     table {
+       border: 1px ${theme.colors.grayscale.light2} solid;
+     }
+     pre {
+       font-size: 11px;
+       padding: 0px;
+       background-color: transparent;
+       border: 0px;
+       line-height: 110%;
+     }
+   `}
+`;
+
+const StackableHeader = styled(Button)<{ top: number }>`
```

Comment:
This header is currently rendered as all caps (probably inherited style from `Button`). Is that what we want?